synapse/skills/synapse-a2a/SKILL.md

```
@@ -0,0 +1,165 @@
+
+```bash
+python3 synapse/tools/a2a.py list [--live]
+```
+
+- `--live`: Only show running agents (auto-cleanup stale entries)
+
+### Cleanup Stale Entries
+
+```bash
+python3 synapse/tools/a2a.py cleanup
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)